CONTRIBUTING.md

```diff
@@ -0,0 +1,11 @@
+## Testing
+
+### Scripted
+
+`sbt scripted` runs the scripted tests. Note that for the `sbtix/private-auth` test you will need to have the `./serve-authenticated.py` server running.
```

Comment:
Does this assume a nix shell?

Reply:
Apparently, it assumes the user environment has Python and sbt. 
To be fair, the current status of private repo credentials handling is sub-par. I don't want to invest more in it because we currently don't have the need for it in production.